### ISSUE TITLE
[A11Y] OPS Reader View - broken skip links

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12645]
## Issue Description
Skip links are currently non-functional when activating them via screenreaders.

## How to reproduce / Current behavior
1. Use a screen reader (NVDA/JAWS) to navigate an OPS page.
2. Press `TAB` until reach the `skip to...` links
3. Note that "Skip to main content" links do not move focus to the correct area.

## What application are you using?
OPS version 3.5.0-3

## Stack used 
- Device: Desktop
- OS: Windows
- Browser: Chrome

## Testing method
- Method: Manual test using NVDA or JAWS

## Action
1. Fix skip link targets to ensure they move focus to the `<main>` content.

## Markup

The link component is empty (no label)
`<a id="pkp_content_main"></a>`
This is an empty anchor used as a fragment target . Historically this worked fine. but more recently screen readers like JAWS and NVDA rely heavily on focusable elements and semantic landmarks.

This could be fixed moving the ID to a landmark element, e.g.;
```
<main id="pkp_content_main">
  ...
</main>
``` 

### WCAG Success Criteria
2.4.1 Bypass Blocks (A)
2.4.4 Link Purpose (In Context) (A)
